purchase_order_line_payment_forecast/models/purchase_order_line.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import api, fields, models
+
+
+class PurchaseOrderLine(models.Model):
+    _inherit = "purchase.order.line"
+
+    price_subtotal_billed = fields.Monetary(
+        compute="_compute_price_subtotal", store=True
```

Comment:
To make the scope clearer.
```suggestion
        compute="_compute_price_subtotal_billed", store=True
```